### DISCUSSION TITLE
Loselesscut not showing whole video in timestamp

### DISCUSSION THREAD

**Discussion by roel2103** [mifi/lossless-cut#1814] [Q&A]
I have a video with a duration of 32 hours but when i place the video in loselesscut it only shows max 12 hours and 25 minutes.

Is there a way to show the whole video so i can cut it ?

![1](https://github.com/mifi/lossless-cut/assets/153460084/089eec1d-6b3e-4f7a-a377-eec3c8a010cc)
![2](https://github.com/mifi/lossless-cut/assets/153460084/fdcb8612-748b-425b-b71b-570b01d5da36)

**Comment by mifi** [comment]
Hi. You closed it as resolved. how did you fix it?
